deisctl/units/deis-kube-apiserver.service

```diff
@@ -0,0 +1,30 @@
+[Unit]
+Description=Kubernetes API Server
+Documentation=https://github.com/GoogleCloudPlatform/kubernetes
+Requires=etcd2.service fleet.service docker.service flanneld.service
```

Comment:
We are actually running etcd2 _as_ "etcd.service" still.  This should probably change.